torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -17,6 +17,7 @@
 #include <ATen/core/thread_pool.h>
 #include <c10/util/SmallVector.h>
 
+#include <algorithm>
```

Comment:
What's this header for?

Reply:
Ah, I'll remove it. I needed it for `std::reverse`